ci-operator/step-registry/cert-manager/clusterissuer/cert-manager-clusterissuer-commands.sh

```diff
@@ -90,7 +90,7 @@ metadata:
   name: $CLUSTERISSUER_NAME
 spec:
   acme:
-    server: https://acme-v02.api.letsencrypt.org/directory
+    server: https://acme-staging-v02.api.letsencrypt.org/directory
```

Comment:
@lunarwhite why? For customer usage, they would mostly like to use production acme server which provides trusted certificate. And our job's frequency `f14` is not high.

Reply:
No strong reason. The original concern was rate limits, but I may be overthinking here. It does not seem to be an issue at least for now (f14).